lib/grape-swagger/endpoint/params_parser.rb

```diff
@@ -34,8 +31,8 @@ def parse_request_params
 
       private
 
-      def array_use_braces?(options)
-        settings[:array_use_braces] && !(options[:documentation] && options[:documentation][:param_type] == 'body')
+      def array_use_braces?
+        @array_use_braces ||= settings[:array_use_braces] && !includes_body_param?
```

Comment:
where are the settings come from, ist it configurable?
if yes, then it should be described in the README

Reply:
It's described here: https://github.com/ruby-grape/grape-swagger#array_use_braces